open-api.yaml

```diff
@@ -0,0 +1,1614 @@
+openapi: 3.1.0
+info:
+  title: MeiliSearch
+  description: MeiliSearch API
+  version: 0.21.0
+  contact:
+    name: MeiliSearch
+    email: bonjour@meilisearch.com
+  license:
+    name: MIT
+    url: 'https://github.com/meilisearch/MeiliSearch/blob/main/LICENSE'
+servers:
+  - url: '{protocol}://{domain}:{port}'
+    variables:
+      protocol:
+        enum:
+          - https
+          - http
+        default: https
+      domain:
+        default: example.meilisearch.com
+      port:
+        default: '7700'
+    description: 'https://example.meilisearch.com:7700'
+components:
+  schemas:
+    timestamp:
+      type: string
+      example: '2019-11-20T09:40:33.711324Z'
+    index:
+      type: object
+      additionalProperties: false
+      properties:
+        uid:
+          type: string
+          example: movies
+        primaryKey:
+          type: string
+          nullable: true
+          example: movie_id
+        createdAt:
+          $ref: '#/components/schemas/timestamp'
+        updatedAt:
+          $ref: '#/components/schemas/timestamp'
+      required:
+        - uid
+        - primaryKey
+        - createdAt
+        - updatedAt
+    document:
+      type: object
+      additionalProperties: true
+      example:
+        id: 25684
+        title: American Ninja 5
+        poster: 'https://image.tmdb.org/t/p/w1280/iuAQVI4mvjI83wnirpD8GVNRVuY.jpg'
+        overview: 'When a scientists daughter is kidnapped, American Ninja, attempts to find her, but this time he teams up with a youngster he has trained in the ways of the ninja.'
+        release_date: '1993-01-01'
+      description: A document made of attribute
+    matchesInfo:
+      type: object
+      properties:
+        start:
+          type: integer
+        length:
+          type: integer
+      additionalProperties: false
+      required:
+        - start
+        - length
+      description: ''
+    hit:
+      type: object
+      additionalProperties: true
+      example:
+        id: 25684
+        title: American Ninja 5
+        poster: 'https://image.tmdb.org/t/p/w1280/iuAQVI4mvjI83wnirpD8GVNRVuY.jpg'
+        overview: 'When a scientists daughter is kidnapped, American Ninja, attempts to find her, but this time he teams up with a youngster he has trained in the ways of the ninja.'
+        release_date: '1993-01-01'
+        _formatted:
+          id: 25684
+          title: American Ninja 5
+          poster: 'https://image.tmdb.org/t/p/w1280/iuAQVI4mvjI83wnirpD8GVNRVuY.jpg'
+          overview: 'When a scientists daughter is kidnapped, American <em>Ninja</em>, attempts to find her, but this time he teams up with a youngster he has trained in the ways of the <em>ninja</em>.'
+          release_date: '1993-01-01'
+        _matchesInfo:
+          overview:
+            - start: 49
+              length: 5
+            - start: 155
+              length: 5
+      examples:
```

Comment:
`example` and `examples` keys are present is this on purpose?

Reply:
No! It is generated by the tool. I need to find out why!